lib/src/test/resources/compileContracts.sh

```diff
@@ -16,21 +13,6 @@ echo "Installing scarb..."
 asdf plugin add scarb || true
 echo "Done!"
 
-pushd "$(dirname "$0")" || exit 1
-mkdir -p "$V0_ARTIFACT_PATH"
-echo "Compiling v0 contracts.."
-
-while IFS= read -r -d '' file; do
-  name="$(basename -- "$file" .cairo)"
-  if [[ $name == *"account"* ]]; then
-    starknet-compile-deprecated "$file" --account_contract --output "$V0_ARTIFACT_PATH/$name.json" --abi "$V0_ARTIFACT_PATH/${name}Abi.json" || exit 1
```

Comment:
We can remove `cairo-lang` dependency as we no longer use `starknet-compile-deprecated`.

Since that is the only python dependency, that should allow us to remove steps related to installing python requirements on CI.

Reply:
Removed.